### ISSUE TITLE
New loss function : bayesian loss 

### ISSUE THREAD

**Issue by deebuls** [RnDProjectsDeebul/SathwikPanchangamRnD#21]
https://aleximmer.github.io/Laplace/

With the package u can convert any pytorch model to bayesian model and get uncertainty. 

Please check it .

There is a blog to explain the paper.
https://towardsdatascience.com/go-deep-but-also-go-bayesian-ab25efa6f7b
